### PR TITLE
[1/n] JUnit Migration - Migrate exception assertion to Junit5

### PR DESCRIPTION
**Pull request by chaaransen**
Migrated from expected exception annotation check (Junit4) to assertThrows method (junit 5)

### CODE REVIEW

File: extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java [hunk 1]
```diff
@@ -53,192 +55,180 @@
...
+    });
+  }
+
+  @Test
+  public void testInitErrorCreatingBucket() throws BackblazeCredentialsException, IOException {
+    createEmptyBucketList();
+    when(s3Client.createBucket(any(CreateBucketRequest.class)))
+        .thenThrow(AwsServiceException.builder().build());
+    BackblazeDataTransferClient client = createDefaultClient();
+    Assertions.assertThrows(IOException.class, () -> {
```
Comment (xokker) [inline extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java:143]: Could you please use a static import here and everywhere we do `assertThrows`? So it is in-line with other asserts.

File: extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java [hunk 2]
```diff
@@ -53,192 +57,180 @@
...
-        when(s3Client.uploadPart(any(UploadPartRequest.class), any(RequestBody.class)))
-                .thenThrow(AwsServiceException.builder().build());
-        final long fileSize = testFile.length();
-        BackblazeDataTransferClient client =
-                new BackblazeDataTransferClient(monitor, backblazeS3ClientFactory, fileSize / 2, fileSize / 8);
+      } catch (Exception ex) {
+        verify(monitor, atLeast(1)).info(any());
+        throw ex;
+      }
+    });
```
Comment (xokker) [inline extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java:136]: Could you please remove the try-catch block and move monitor verification outside? Sorry I missed it during the first review :(

File: extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java [hunk 3]
```diff
@@ -53,192 +57,180 @@
...
         client.init(KEY_ID, APP_KEY, EXPORT_SERVICE);
-        // Act
-        client.uploadFile(FILE_KEY, testFile);
-        // Assert: expected exception
-    }
+      } catch (Exception ex) {
+        verify(s3Client, atLeast(1)).close();
+        verify(monitor, atLeast(1)).debug(any());
+        throw ex;
+      }
```
Comment (xokker) [inline extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/common/BackblazeDataTransferClientTest.java:161]: The same here, no need for try-catch

**Review by xokker — COMMENTED** [review]
I am sorry I did not catch it the first time

**Review by xokker — APPROVED** [review]
Thank you for addressing the comments!

**Review by seehamrun — APPROVED** [review]

**Pull request merged**